### PR TITLE
[NEAT-332] 	🧹Cleanup pyproject.toml

### PR DESCRIPTION
**Pull request by doctrino**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -20,12 +20,6 @@ exclude = [
     "cognite/neat/app/ui/neat-app/*.json",
 ]
 
-
-[tool.black]
-line-length = 120
-target_version = ['py310']
-include = '\.py$'
```
Comment (doctrino) [inline pyproject.toml:27]: Forgotten, we stopped using black a long time ago

File: pyproject.toml [hunk 2]
```diff
@@ -55,36 +49,46 @@ warn_required_dynamic_aliases = true
...
-schedule = "^1"
-python-multipart = "^0"
-oxrdflib = { version = "^0.3.3", optional = true, extras = ["oxigraph"] }
+# API
+uvicorn = { extras = ["standard"], version = "^0", optional = true }
+prometheus-client = {version="^0", optional = true}
+fastapi = {version="^0", optional = true }
 
+# WorkFlow
+schedule = {version ="^1", optional = true}
```
Comment (doctrino) [inline pyproject.toml:77]: Idea is to not force python users to run the service.

File: pyproject.toml [hunk 3]
```diff
@@ -112,14 +115,17 @@ docs = [
...
 google = ["gspread", "google-api-python-client", "google-auth-oauthlib"]
-oxi = ["oxrdflib"]
-graphql = ["jinja2"]
+oxi = ["oxrdflib", "pyoxigraph"]
+service = ["uvicorn", "prometheus-client", "fastapi", "schedule"]
+
 all = [
-    "gspread",
-    "google-api-python-client",
-    "google-auth-oauthlib",
```
Comment (doctrino) [inline pyproject.toml:120]: I removed these from all as they are only needed for a very specific case. Then, I think the user should install them explicitly `pip install neat[google]`.

File: pyproject.toml [hunk 4]
```diff
@@ -132,8 +138,6 @@ pytest-notebook = "*"
 freezegun = "^1.5.1"
 pytest-xdist = "^3.6.1"
 pre-commit = "*"
-safety = "*"
-bandit = "*"
```
Comment (doctrino) [inline pyproject.toml:136]: Not used.

File: pyproject.toml [hunk 5]
```diff
@@ -55,36 +49,46 @@ warn_required_dynamic_aliases = true
...
 typing_extensions = { version = "^4.8", python = "<3.11" }
 tomli = { version = "^2.0.1", python = "<3.11" }
 
-uvicorn = { extras = ["standard"], version = "^0" }
-prometheus-client = "^0.20.0"
-cognite-sdk = "^7.37.0"
-deepdiff = "*"
-fastapi = "^0"
-schedule = "^1"
-python-multipart = "^0"
```
Comment (doctrino) [inline pyproject.toml:80]: Not used as far as I can see.

**Review by nikokaoja — APPROVED** [review]

**Pull request merged**